### PR TITLE
Toggle users page

### PR DESCRIPTION
**Pull request by AsmaaELM**
Issue: https://scality.atlassian.net/browse/ZKUI-104

### CODE REVIEW

**Comment by AsmaaELM** [comment]
ping

File: src/react/account/AccountDetails.tsx [hunk 1]
```diff
@@ -18,9 +20,19 @@ const NotFound = () => (
...
 );
 
+declare module 'styled-components' {
+  export interface DefaultTheme {
+    brand: {
+      backgroundLevel2: string,
+      backgroundLevel3: string
+    };
+  }
+}
```
Comment (JBWatenbergScality) [inline src/react/account/AccountDetails.tsx]: This type should likely be global to the project and hold all the theme fields

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**